File: Northwind_Demo/Sources/NorthWind.Web.Angular/src/app/domain/wizards/initialization/orderDetailWizardInitialization.js
import CodeBehindOrderDetailWizardInitialization from './codeBehind/codeBehindOrderDetailWizardInitialization';

export default class OrderDetailWizardInitialization extends CodeBehindOrderDetailWizardInitialization {
	constructor(
		//--  custom dependencies
		//-- /custom dependencies
		httpService){
		"ngInject";
		super(httpService);
	}
}

File: Northwind_Demo/Sources/NorthWind.Common/Wizards/Initialization/OrderDetailWizardInitialization.cs
using System;
using BusinessFramework.Client.Contracts.Wizards;
using NorthWind.Client.Contracts.BusinessObjects;


namespace NorthWind.Common.Wizards.Initialization
{
    public sealed class OrderDetailWizardInitialization : CodeBehind.CodeBehindOrderDetailWizardInitialization
    {
        /// <summary>
        /// Ctor
        /// </summary>
        public OrderDetailWizardInitialization(
        //--  custom dependencies
		//-- /custom dependencies
        )
            :base()
        {
        }
    }
}